app/page.tsx

```diff
@@ -17,6 +18,7 @@ export default async function Home() {
           <Header />
           <Searchbar />
           <HeaderMenu />
+          <Homepage />
           <div className="main--content"></div>
```

Comment:
Je pense que le composant Homepage devrait se trouver dans cette div ```main--content```

```suggestion
          <div className="main--content">
             <Homepage />
          </div>
```